docs/index.md

```diff
@@ -26,23 +44,40 @@ terraform {
   }
 }
 
-# Configure the Civo Provider
 provider "civo" {
-  token = var.civo_token
   region = "LON1"
 }
 
-# Create a web server
-resource "civo_instance" "web" {
-  # ...
+```
+
+### Example with credentials file
+
+In this example we are providing a specific version of the terraform provider and setting a credentials file to use. The credentials file will be used if the environment variable `CIVO_TOKEN` is not set.
```

Comment:
we should also display contents of a dummy credentials file

Reply:
So my understanding is that the credentials file can be a file with just the token, or using the `.civo.json` format. But we would not expect an end user to use the latter for this file, correct?